modules/ossm-cr-istio-global-1x.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+[id="ossm-cr-istio-global_{context}"]
+= Istio global example
+
+Here is an example that illustrates the Istio global parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
+
+[NOTE]
+====
+In order for the 3scale Istio Adapter to work, `disablePolicyChecks` must be `false`.
+====
+
+.Example global parameters
```

Comment:
Do we want to update this to smcp v2?  If so: (although, I don't think we should include imageTag and imageRegistry)

```yaml
spec:
  runtime:
    defaults:
      container:
        imageTag: 1.1.0
        imageRegistry: registry.redhat.io/openshift-service-mesh
        imagePullSecrets:
        - name: MyPullSecret
  proxy:
    runtime:
      container:
        resources:
          requests:
            cpu: 10m
            memory: 128Mi
          limits: {}
  security:
    dataPlane:
      mtls: true
  policy:
    mixer:
      enableChecks: false
      failOpen: false
```

Reply:
Also, do we want to just redirect this to the smcp v2 docs?  They apply to all versions of installations, with some minor nuances.

Reply:
I think we shouldn't update it to smcpv2. I think we should keep this as smcp v1 in the v1 docs.